pkg/clierrors/clierrors.go

```diff
@@ -0,0 +1,11 @@
+// Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package clierrors
```

Comment:
is it better to place this under pkg/constants/error.go?

Reply:
im fine either way

Reply:
lets keep it here but happy to hear @learyce opinion

Reply:
I'd say constants

Reply:
moving into constants